src/index.json

```diff
@@ -17044,9 +17044,9 @@
                         }
                     ],
                     "summary": "Tools for managing Azure DevOps.",
-                    "version": "0.26.0"
```

Comment:
I'm curious why you update the entry for `0.26.0`, usually we need add a new item for `0.27.0`.

Reply:
@jsntcy This was the new release version we published on devops extension. Below is the last PR we made changes as same to release new version.
https://github.com/Azure/azure-cli-extensions/pull/5803

Reply:
@jsntcy any update on this??

Reply:
You should not overwrite metadata of `0.26.0` for new version `0.27.0`, because if you did like that, cx cannot install `0.26.0` any longer once you release `0.27.0`. You should add a new entry for `0.27.0` like all other extensions did, an example: https://github.com/Azure/azure-cli-extensions/commit/5ce810ed7c3c6ce7714a806537527a01dd5764d0.

Reply:
BTW, 
Now we support new versioning for extension and 0.x.x is not recommended any longer. The version will start from 1.0.0 (stable) or 1.0.0b1 (preview). You can find more details in  https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md.
Please update the version to correct one next time.

Reply:
HI @jsntcy, our new version is 0.27.0 as per the new version guidelines we are making it as 1.27.0. Please confirm is it okay?? post your confirmation I will create a new entry with the new version.

Reply:
Hi @jsntcy  Any suggestions on above ask by Soujanya from your end? We need to do some more changes with the new rule conventions so taking suggestions here first.

Reply:
> HI @jsntcy, our new version is 0.27.0 as per the new version guidelines we are making it as 1.27.0. Please confirm is it okay?? post your confirmation I will create a new entry with the new version.

If it's stable version, we recommend you use version `1.0.0` as it's first version from`1.x.x`.